<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="map_d">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>781</width>
      <height>351</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="adress_d">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>181</y>
      <width>781</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>781</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="place_input">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="find_b">
       <property name="text">
        <string>Искать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_b">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="index_b">
       <property name="text">
        <string>Показать индекс</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1</y>
      <width>781</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="satellite_rb">
       <property name="text">
        <string>Спутник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="hybrid_rb">
       <property name="text">
        <string>Гибрид</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="scheme_rb">
       <property name="text">
        <string>Схема</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
